### DISCUSSION TITLE
Provide documentation of how this differs from other lsposed forks

### DISCUSSION THREAD

**Discussion by Jordan231111** [mywalkb/LSPosed_mod#74] [General]
### Is your feature request related to a problem?

Unclear what is different, and how its different from other versions such as Pumclin fork of lsposed

### Describe the solution you'd like

Add in Wiki tab or somewhere you think is good for users to be informed

**Comment by mywalkb** [comment]
Just read the README.
I wrote them because I started the _mod fork long before the initial project closed and I wrote them everything it does.
However I will try to improve it.
I don't know the other forks and don't have the time to study them.
